src/test/java/com/ays/admin_user/controller/AdminUserRegisterApplicationControllerTest.java

```diff
@@ -281,4 +281,58 @@ void givenValidAdminUserRegisterApplicationId_whenAdminUserApplicationFound_then
                         .isNotEmpty());
     }
 
+    @Test
+    void givenValidAdminUserRegisterApplicationId_whenApproveAdminUserRegisterApplication_thenReturnNothing() throws Exception {
+        // Given
+        String mockId = AysRandomUtil.generateUUID();
+
+        // When
+        Mockito.doNothing().when(adminUserRegisterApplicationService).approveRegistrationApplication(mockId);
+
+        // Then
+        String endpoint = BASE_PATH.concat("/registration-application/".concat(mockId).concat("/approve"));
+        AysResponse<Void> mockAysResponse = AysResponse.SUCCESS;
+
+        mockMvc.perform(AysMockMvcRequestBuilders
+                        .post(endpoint, mockSuperAdminToken.getAccessToken()))
+                .andDo(MockMvcResultHandlers.print())
+                .andExpect(AysMockResultMatchersBuilders.status().isOk())
+                .andExpect(AysMockResultMatchersBuilders.time()
+                        .isNotEmpty())
+                .andExpect(AysMockResultMatchersBuilders.httpStatus()
+                        .value(mockAysResponse.getHttpStatus().getReasonPhrase()))
+                .andExpect(AysMockResultMatchersBuilders.isSuccess()
+                        .value(mockAysResponse.getIsSuccess()))
+                .andExpect(AysMockResultMatchersBuilders.response()
+                        .doesNotExist());
+
+        Mockito.verify(adminUserRegisterApplicationService, Mockito.times(1))
```

Comment:
Aşağıdaki gibi suggestion kısmını da ekleyebilir miyiz?
```suggestion
        // Verify
        Mockito.verify(adminUserRegisterApplicationService, Mockito.times(1))
```